src/Libraries/Microsoft.Extensions.Http.Resilience/Polly/HttpRetryStrategyOptions.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Http.Resilience.Internal;
+using Polly.Retry;
+
+namespace Microsoft.Extensions.Http.Resilience;
+
+/// <summary>
+/// Implementation of the <see cref="RetryStrategyOptions{TResult}"/> for <see cref="HttpResponseMessage"/> results.
+/// </summary>
+public class HttpRetryStrategyOptions : RetryStrategyOptions<HttpResponseMessage>
+{
+    private bool _shouldRetryAfterHeader;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="HttpRetryStrategyOptions"/> class.
+    /// </summary>
+    /// <remarks>
+    /// By default, the options are set to handle only transient failures,
+    /// that is, timeouts, 5xx responses, and <see cref="HttpRequestException"/> exceptions.
+    /// </remarks>
+    public HttpRetryStrategyOptions()
+    {
+        ShouldHandle = args => new ValueTask<bool>(HttpClientResiliencePredicates.IsTransientHttpOutcome(args.Outcome));
+        BackoffType = RetryBackoffType.ExponentialWithJitter;
+        ShouldRetryAfterHeader = true;
+    }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether to retry after header.
```

Comment:
This sentence doesn't really make any sense. Did you mean _"...whether to use the Retry-After header"_?

Reply:
True, fixed.